github_connector/models/github_repository_branch.py

```diff
@@ -275,8 +291,65 @@ def _analyze_code(self):
                         branch.name,
                         e,
                     )
+
         return True
 
+    def _prepare_analysis_rule_model_info(self, analysis_rule_id):
+        """
+        Define model data info that override with other addons
+        """
```

Comment:
In this case docstring fits in one line, so it should be:
```suggestion
        """Define model data info that override with other addons."""
```